huxley/invoice_automation/src/handler/fc_handler.py

```diff
@@ -0,0 +1,112 @@
+import os.path
+import re
+import datetime
+from typing import List
+
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+# If modifying these scopes, delete the file token.json.
+from huxley.invoice_automation.src import handler
+from huxley.invoice_automation.src.model.address import Address
+from huxley.invoice_automation.src.model.conference import Conference
+from huxley.invoice_automation.src.model.registration import Registration
+from huxley.invoice_automation.src.model.school import School
+
+SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly']
+
+# The ID and range of a sample spreadsheet.
+SAMPLE_SPREADSHEET_ID = '1w-Xt-LmWnWvmzDJ6EAp-_I6WCJe9XDlHyAcWOsZ0ZWI'
+SAMPLE_RANGE_NAME = 'Responses for Invoicing!A2:K'
+
+TIMESTAMP_REGEX = r"(\d{1,2})\/(\d{1,2})\/(\d{4})"
+
+
+def main():
+    """Shows basic usage of the Sheets API.
+        Prints values from a sample spreadsheet.
+        """
+    creds = None
+    # The file token.json stores the user's access and refresh tokens, and is
+    # created automatically when the authorization flow completes for the first
+    # time.
+    token_path = '../../google_auth_resources/token.json'
```

Comment:
can you move this to django settings similar to how the sheets integration already does it?

Reply:
Honestly I'll remove fc_handler from the commit, since it's not actually relevant to Huxley.

Reply:
sg